automatic/git.install/git.install.nuspec

```diff
@@ -29,7 +29,7 @@ Git for Windows focuses on offering a lightweight, native set of tools that brin
 
 - The package uses default install options minus cheetah integration and desktop icons. Cheetah prevents a good upgrade scenario, so it has been removed.
 - For disabling the Git Credential Manager see the [Disable Git Credential Manager package](https://chocolatey.org/packages/git-disable-gcm).
```

Comment:
@pascalberger ignore the description inside the nuspec file, have a look at the Readme.md instead.
You see there that this sentence is removed (and if you look at the update.ps1 file, the description gets updated from the Readme.md file).